e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -114,9 +114,18 @@ export function saveDashboard({
   buttonLabel = "Save",
   editBarText = "You're editing this dashboard.",
   waitMs = 1,
+  awaitRequest = true,
 } = {}) {
+  cy.intercept("PUT", "/api/dashboard/*").as("saveDashboardCards");
   cy.button(buttonLabel).click();
-  cy.findByText(editBarText).should("not.exist");
+
+  if (awaitRequest) {
```

Comment:
Some e2e tests may fail due to this change, I'll have to take a look at the failures then update them